docs/devel/api_changes.md

```diff
@@ -485,6 +485,17 @@ To regenerate them:
 hack/update-generated-deep-copies.sh
 ```
 
+Since our components still use internal apis, you may need to generate code
```

Comment:
Or actually merge it with the above - we no longer have update-generate-deep-copies